libs/common-services/src/main/java/uk/gov/di/ipv/core/library/helpers/LogHelper.java

```diff
@@ -139,7 +147,14 @@ public static StringMapMessage buildErrorMessage(String message, String errorDes
     }
 
     public static StringMapMessage buildErrorMessage(String message, Exception e) {
-        return buildLogMessage(message).with(LOG_ERROR_DESCRIPTION.getFieldName(), e);
+        var mapMessage = buildLogMessage(message).with(LOG_ERROR_DESCRIPTION.getFieldName(), e);
+        if (LOGGER.isDebugEnabled()) {
```

Comment:
I'm trying to work out if this is confusing or not. If we want to get the stacktrace we need to set the `LogHelper` logger to debug level. But it's not the actual logger doing the logging - it's just being used as a flag. It does give an easy way to just target logging of these stack traces from any logger, rather than having to set a bunch of different loggers (or everything) to debug. But actually setting it to debug is a bit of a pain. If this is the behaviour we want, could it be done with a feature set?

Reply:
Config to add to log4j2.json that turned on debugging, added to Logging.loggers
```
        {
          "name": "uk.gov.di.ipv.core.library.helpers.LogHelper",
          "level": "debug",
          "additivity": false,
          "AppenderRef": {
            "ref": "JsonAppender"
          }
        }
```

Reply:
As discussed, going to revert this for now. I still think stack traces are important, but not related to this change.